lib/resolvers.js

```diff
@@ -38,6 +70,56 @@ const Resolvers = {
     }
   },
   Mutation: {
+    createAccount: async (root, { id, isAdmin }, request) => {
+      await validateAdminAccess(request);
+      const sql = 'CALL create_account(?, ?)';
+      const pool = request.server.app.mysql;
+      const results = await Common.doQuery(sql, [id, isAdmin ? 1 : 0], pool);
+
+      // Sanity check. This should always be one.
+      if (results.affectedRows !== 1) {
+        throw Boom.internal(`Insertion affected ${results.affectedRows} accounts`);
+      }
+
+      return { id, isAdmin };
+    },
+
+    updateAccount: async (root, { id, isAdmin }, request) => {
+      await validateAdminAccess(request);
+      const sql = 'CALL update_account(?, ?)';
+      const pool = request.server.app.mysql;
+      const results = await Common.doQuery(sql, [id, isAdmin ? 1 : 0], pool);
+
+      // Sanity check. This should always be one.
+      if (results.affectedRows !== 1) {
+        throw Boom.internal(`Update affected ${results.affectedRows} accounts`);
+      }
+
+      return { id, isAdmin };
+    },
+
+    deleteAccount: async (root, { id }, request) => {
+      await validateAdminAccess(request);
+
+      const getSql = 'CALL get_account(?)';
+      const pool = request.server.app.mysql;
+      const [getResults] = await Common.doQuery(getSql, [id], pool);
```

Comment:
Doing the get and then delete in JS seems racey. You could do these as a transaction in SQL with one less trip to the db.